pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
Seems like apache_beam/runners is for runner implementations. So I think it makes sense to add apache_beam/core/runners and move apache_beam/runners/api (and apache_beam/runners/portability) to there. We should prob. also move transforms/core.py (and other core modules) to apache_beam/core but we won't be able to do that till next major version.

cc: @aaltay @sb2nov